Histogram.py

```diff
@@ -0,0 +1,169 @@
+from coffea.nanoevents import NanoEventsFactory, DelphesSchema
+
+def ET(vec):
+    return np.sqrt(vec.px**2+vec.py**2+vec.mass**2)
+
+def get_values(var,Events):
+    return ak.flatten(Events[var],axis=None)
+
+def fill_hist(var,nbins,bmin,bmax,label,Events):
+    h = (
+        hist.Hist.new
+        .Reg(nbins, bmin, bmax, label=label)
+        .Double()
+    )
+    h.fill(get_values(var,Events),weight=0.5)
+    return h
+
+def normalize_angle(angle):
+    angle = np.mod(angle, 2 * np.pi)
+    angle = np.where(angle >= np.pi, angle - 2 * np.pi, angle)
+    return angle
+
+def histogram(filename, model_name):                    
+    DelphesSchema.mixins["FatJet"] = "Jet"
+    Events = NanoEventsFactory.from_root(
+                file=filename,
+                    treepath="Delphes",
+                        schemaclass=DelphesSchema,
+                        ).events()
+    
+    # require two jets
+    events = Events
+    mask = ak.num(events.FatJet)>=2
+    events = events[mask]
+
+    # Dijet
+    events["Dijet"] = events.FatJet[:,0]+events.FatJet[:,1]
+
+    # transverse mass calculation
+    E1 = ET(events.Dijet)
+    E2 = events.MissingET.MET
+    MTsq = (E1+E2)**2-(events.Dijet.px+events.MissingET.px)**2-(events.Dijet.py+events.MissingET.py)
+    events["MT"] = np.sqrt(MTsq, where=MTsq>=0)
+            
+    # 4-vectors for dijet
+    events["Jet_pt"] = events.Dijet.pt
+    events["Jet_eta"] = events.Dijet.eta
+    events["Jet_phi"] = events.Dijet.phi
+    events["Jet_mass"] = events.Dijet.mass
+    # Other quantites to be plotted
+    events["MET"] = events.MissingET.MET
+    events["DeltaEta"] = events.FatJet.DeltaEta
+    events["DeltaPhi"] = events.FatJet.DeltaPhi
+
+    ## For plotting individually for jet1 and jet2
+    leading_jets = events.FatJet[:, :2]
+
+    jet1_mask = leading_jets[:, 0].pt >= leading_jets[:, 1].pt
+    jet2_mask = ~jet1_mask
+
+    events["Jet1"] = ak.where(jet1_mask, leading_jets[:, 0], leading_jets[:, 1])
+    events["Jet2"] = ak.where(jet2_mask, leading_jets[:, 0], leading_jets[:, 1])
+
+    # 4-vectors for jet1 and jet2
+    events["Jet1_pt"] = events["Jet1"].pt
+    events["Jet2_pt"] = events["Jet2"].pt
+    
+    events["Jet1_eta"] = events["Jet1"].eta
+    events["Jet2_eta"] = events["Jet2"].eta
+    
+    events["Jet1_phi"] = events["Jet1"].phi
+    events["Jet2_phi"] = events["Jet2"].phi
+    
+    events["Jet1_mass"] = events["Jet1"].mass
+    events["Jet2_mass"] = events["Jet2"].mass
+    
+    events["DeltaEta"] = np.abs(events["Jet2_eta"] - events["Jet1_eta"])
+    events["DeltaPhi"] = np.abs(normalize_angle(events["Jet1_phi"] - events["Jet2_phi"]))
+    
+    events["DeltaPhi_MET_Jet1"] = np.abs(normalize_angle(events.MissingET.phi - events["Jet1_phi"]))
+    events["DeltaPhi_MET_Jet2"] = np.abs(normalize_angle(events.MissingET.phi - events["Jet2_phi"]))
+
+    # Stable inv frac
+    dark_hadron_ids = [4900111, 4900113, 4900211, 4900213]
+    stable_particle_ids = [51, 52, 53]
```

Comment:
The helper now has member variables for these: `helper.darkHadronIDs`, `helper.stableIDs`. You should pass the instance of the helper from `run_model` into this function and then get these lists from it.